frontend/src/components/WaitingRoom/PreviewAvatar/PreviewAvatar.tsx

```diff
@@ -26,6 +28,10 @@ const PreviewAvatar = ({
   isVideoEnabled,
   isVideoLoading,
 }: PreviewAvatarProps): ReactElement | null => {
+  const smallDisplayDimensions = useWindowWidth() * 0.46;
```

Comment:
Just curious, how was this magic literal arrived at?

Reply:
A bit of trial and error, but it seemed to fit well with our minimum supported width of `360px`. Scaling up to a normal desktop viewport (see the gif), the avatar-initials looked good, too.